### ISSUE TITLE
[Feature][Idea] Ingame Character Preview

### ISSUE THREAD

**Issue by Drakray** [Kitteh6660/Corruption-of-Champions-Mod#189]
I think it would be SUPER AWESOME to have a preview included in the game, as all the external tools are REALLY outdated
Could be an option, for people who want to picture their character themselves.
using a handful of vector image snapped together and using game value to enlarge/shrink feature could even allow for "texture pack"
but I understand that this is REALLY big and really not a priority, if it ever come to happen lol :P
would just be nice as it would give reason to "customize" the character, like hair/fur color and such
Preview would be in Appearance tab, for obvious reason

**Comment by Killette2** [comment]
I don't believe a character viewer for a text based game is as good of an idea as you may think. The purpose of most scenes is to use your imagination to picture them, and that is also why character sprites are pixelated to leave as much to the imagination as possible while still showing general color schemes and body figure. Nothing would kill my boner harder than seeing my sexy character made in a flash generated deviantart tier picture, that could never beat the power of imagination.

**Comment by Drakray** [comment]
I understand, that's why I think it should only be an option, like silly mode and such
I think it would be great even in a text based game, mainly because of all the aesthetic customization
But IF it is implemented, it will probably be by a separate team, included as a pull request, and manage by exterior people, like some version of the old Character Viewer

**Comment by Killette2** [comment]
I guess that makes sense, but I have the feeling curiosity will get to me and I'll look anyways.

On the other hand, this would enable me to make abominations...

**Comment by Drakray** [comment]
Depending on how it's made, it could even support "artist pack", to get different different style, something like the actual system.
I think a list of vector image part, following a basic template, then rendered a certain way so as long as the pack follow the template, it should render correctly.
I think there would be people to fill the art part, I know I would, if I was not so bad with visual art XD

**Comment by drakray** [comment]
It's coming alive!! :D
Quick question, how do I make use of the feature?
I enabled it in option/Interface, but ingame, nothing shows in Apperance.
